subsys/mgmt/ec_host_cmd/Kconfig

```diff
@@ -17,12 +17,21 @@ if EC_HOST_CMD
 
 config EC_HOST_CMD_HANDLER_STACK_SIZE
 	int "Stack size for the EC host command handler thread"
-	default 512
+	default 800
 
 config EC_HOST_CMD_HANDLER_TX_BUFFER
 	int "Buffer size in bytes for TX buffer shared by all EC host commands"
 	default 256
 
+config EC_HOST_CMD_HANDLER_PRIO
+	int "Priority of host command task"
+	default -16
+	help
+	  Priority of the kernel task that handles the host commands.
+	  Default value is the highest priority based on the default value of
+	  cooperative tasks number. If the priority is too low, the host
```

Comment:
Well this would not be true anymore if NUM_COOP_PRIORITIES changes. Do we need it to be configurable? Sounds like we have a hard requirement, I'd say let's just hardcode it, it's done in other places of the codebase anyway, stuff like

`#define THREAD_PRIORITY K_PRIO_COOP(CONFIG_NUM_COOP_PRIORITIES - 1)`

Reply:
The EC application doesn't use any cooperative threads, so this effectively makes the EC host command handler the highest priority - it will block all other threads while it is running.

Reply:
The high priority was needed due to other bug, I have found an issue with semaphores. Now it works with lower ones.
Anyway, it should still be configurable, as we have our fixed priorities for tasks, and our tasks priorities are outside of Zephyr code. And as for default, I chose value suitable for that moment, I can change it to 0 or some other value if you prefer.